bindings_node/src/mls_client.rs

```diff
@@ -0,0 +1,209 @@
+use std::collections::HashMap;
+use std::ops::Deref;
+use std::sync::Arc;
+
+use crate::conversations::NapiConversations;
+use napi::bindgen_prelude::{Error, Result, Uint8Array};
+use napi_derive::napi;
+use xmtp_api_grpc::grpc_api_helper::Client as TonicApiClient;
+use xmtp_cryptography::signature::ed25519_public_key_to_address;
+use xmtp_id::associations::{Erc1271Signature, RecoverableEcdsaSignature};
+use xmtp_mls::builder::ClientBuilder;
+use xmtp_mls::identity::IdentityStrategy;
+use xmtp_mls::storage::{EncryptedMessageStore, EncryptionKey, StorageOption};
+use xmtp_mls::Client as MlsClient;
+
+pub type NapiXmtpClient = MlsClient<TonicApiClient>;
```

Comment:
My suggestion would be to prefix objects passed directly across the boundary to be prefixed with `Napi`, and objects that live only in Rust with `Rust`. That's similar to the naming scheme we used in the FFI bindings. In this case, this is a rust-only object.